packages/fxa-auth-server/pm2.config.js

```diff
@@ -24,6 +24,24 @@ module.exports = {
         FORCE_PASSWORD_CHANGE_EMAIL_REGEX: 'forcepwdchange',
         CONFIG_FILES: 'config/secrets.json',
         PORT: '9000',
+        REDIS_HOST: 'localhost',
+        ACCESS_TOKEN_REDIS_HOST: 'localhost',
+        REFRESH_TOKEN_REDIS_HOST: 'localhost',
+        OAUTH_URL: 'http://localhost:9000',
+        AUTH_SERVER_URL: 'http://localhost:9000',
+        MYSQL_HOST: 'localhost',
+        PUBLIC_URL: 'http://localhost:9000',
+        HTTPDB_URL: 'http://localhost:9000',
+        CUSTOMS_SERVER_URL: 'http://localhost:7000',
+        HOST: 'localhost',
+        EMAIL_SERVICE_HOST: 'localhost',
+        MAILER_HOST: 'localhost',
+        SMTP_HOST: 'localhost',
+        HOST_INTERNAL: 'localhost',
+        DD_AGENT_HOST: 'localhost',
+        SYNC_TOKENSERVER_URL: 'http://localhost:5000/token',
+        MEMCACHE_METRICS_CONTEXT_ADDRESS: 'localhost:11211',
```

Comment:
wouold be nice to alphabetize these, now that they are getting longer, but not essential